code/.storybook/preview.tsx

```diff
@@ -312,4 +314,11 @@ export const globalTypes = {
       ],
     },
   },
+  foo: { defaultValue: 'fooDefaultValue' },
+  bar: { defaultValue: 'barDefaultValue' },
+};
+
+export const initialGlobals = {
+  foo: 'fooValue',
+  baz: 'bazValue',
 };
```

Comment:
@tmeasday Can I get an explainer why `defaultValue: 'fooDefaultValue'` should ever be used?
How are initialGlobals.foo and globalTypes.foo.defaultValue connected?

At first glance, me and @valentinpalkovic both concluded that if you set a `initialGlobal.foo`, then `globalTypes.foo.defaultValue` should never be used?

Reply:
I was able to figure it out!